esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -878,8 +879,13 @@ bool EspHttpBinding::doAuth(IEspContext* ctx)
 {
     if(m_authtype.length() == 0 || stricmp(m_authtype.str(), "Basic") == 0)
     {
-        CumulativeTimer* timer = ctx->queryTraceSummaryCumulativeTimer(LogNormal, "custom_fields.basicAuthTime", TXSUMMARY_GRP_ENTERPRISE);
-        CumulativeTimer::Scope authScope(timer);
+        static const char* timerName = "custom_fields.basicAuthTime";
+        CumulativeTimer* timer = ctx->queryTraceSummaryCumulativeTimer(LogNormal, timerName, TXSUMMARY_GRP_ENTERPRISE);
```

Comment:
These PRs are going to need rebasing in the right order as the others are merged.

Reply:
Yes. The first two PRs share a change, and are logically stacked based on complexity (first in the stack is the first to complete the review). This one was stacked out of convenience, before it's priority was changed. The TxSummary change is more important, so my priority is for it to be merged first.